vcbuild.bat

```diff
@@ -36,6 +36,10 @@ set release_urls_arg=
 set build_release=
 set enable_vtune_arg=
 set configure_flags=
+set build_addons=
+
+:: assign path to node_exe
+set node_exe="%config%\node.exe"
```

Comment:
The correct place for this is after all `set config=` have happened, before the `configure_flags` are prepared, exactly at https://github.com/nodejs/node/blob/089c6a4fbaa4f0c737ad1ca09b8f823459983d3a/vcbuild.bat#L93 . @blobor can you move this?